content/en/certificates.md

```diff
@@ -3,78 +3,102 @@ title: Chain of Trust
 linkTitle: Chain of Trust (Root and Intermediate Certificates)
 slug: certificates
 top_graphic: 5
-lastmod: 2020-02-07
+lastmod: 2020-08-02
 ---
 
 {{< lastmod >}}
 
 # Root Certificates
 
 Our roots are kept safely offline. We issue end-entity certificates to subscribers from the intermediates in the next section.
+For additional compatibility as we submit our new Root X2 to various root programs, we have also cross-signed it from Root X1.
 
 * Active
-  * [ISRG Root X1 (self-signed)](/certs/isrgrootx1.pem.txt)
+  * ISRG Root X1 (RSA 4096)
+    * [self-signed](/certs/isrgrootx1.pem.txt)
+  * ISRG Root X2 (ECDSA P-384)
+    * [self-signed](/certs/isrg-root-x2.pem.txt)
+    * [cross-signed by ISRG Root X1](/certs/isrg-root-x2-cross-signed.pem.txt)
 
 We've set up websites to test certificates chaining to our roots.
 
-* ISRG Root X1 Valid Certificate
-  * [https://valid-isrgrootx1.letsencrypt.org/](https://valid-isrgrootx1.letsencrypt.org/)
-* ISRG Root X1 Revoked Certificate
-  * [https://revoked-isrgrootx1.letsencrypt.org/](https://revoked-isrgrootx1.letsencrypt.org/)
-* ISRG Root X1 Expired Certificate
-  * [https://expired-isrgrootx1.letsencrypt.org/](https://expired-isrgrootx1.letsencrypt.org/)
+* ISRG Root X1
+  * [Valid](https://valid-isrgrootx1.letsencrypt.org/)
+  * [Revoked](https://revoked-isrgrootx1.letsencrypt.org/)
+  * [Expired](https://expired-isrgrootx1.letsencrypt.org/)
+* ISRG Root X2
+  * [Valid](https://valid-isrgrootx2.letsencrypt.org/)
+  * [Revoked](https://revoked-isrgrootx2.letsencrypt.org/)
+  * [Expired](https://expired-isrgrootx2.letsencrypt.org/)
 
 # Intermediate Certificates
 
-Under normal circumstances, certificates issued by Let’s Encrypt will come from “Let’s Encrypt Authority X3”. The other intermediate, “Let’s Encrypt Authority X4”, is reserved for disaster recovery and will only be used should we lose the ability to issue with “Let’s Encrypt Authority X3”. We do not use the X1 and X2 intermediates any more.
+Under normal circumstances, certificates issued by Let's Encrypt will come from "Let's Encrypt Authority X3".
```

Comment:
Let's add "or "R3".

Reply:
Do we plan to be issuing from X3 and R3 at the same time? Do you mean "or E1"?

I was under the impression that the order of operations would be:
1) Issue new certs (and land this website update)
2) Start issuing ECDSA from E1
3) Switch RSA issuance from X3 to R3 (and update this page again to show X3/X4 as retired)
Are we planning to do (3) before (2)?

Reply:
Yes, we're planning to do (3) before (2).

(2) is a little more involved than I originally realized, because it effectively puts everyone who issues ECDSA leaves immediately into the "chains to ISRG Root X1" bucket, and with no alternative chain they can serve. It also means people have to use long chains, since ISRG Root X2 isn't yet trusted anywhere.

So what I'd like to do is:
 - Switch to issuing from R3 soon
 - Switch to serving a default chain ISRG Root X1 -> R3 -> EE on Sep 30

(optional: maybe collapse these steps and do them both on Sep 30, since they're each potentially a little disruptive on their own)

As for this specific diff: I wanted to indicate "X3 or R3" because I don't want this page to be the documentation of which specific intermediate we're using at any given time, since that can change on a time scale different than how often people check this page. The only reason we name X3 here is to contrast it with X4 and explain that X4 is a backup and isn't used in normal operations. Maybe there's a better way we can say that?

Reply:
Cool, done. I've pulled R3 into the first line, saying we can issue from either. I've made the next line explicitly about ECDSA, noting that we have an intermediate but aren't issuing from it yet.